_providers/freenet.de.md

```diff
@@ -14,4 +14,13 @@ server:
   type: smtp
 last_checked: 2017-06
 website: https://www.freenet.de/index.html
+before_login_hint: To log in to freenet.de with Delta Chat, you need to activate POP3/IMAP/SMTP in the freenet.de web interface first.
 ---
+
+To use a freenet.de email address with Delta Chat, you need to activate
+POP3/IMAP/SMTP first in the web interface.
+
+You can find it at the "Postfach-Einstellungen", in the category
+"POP3/IMAP/SMTP" at "Einrichtung POP3/IMAP".  Here you need to activate the
+checkbock „Jetzt POP3, IMAP und SMTP aktivieren“ and click on "Speichern"
```

Comment:
checkbox

Reply:
Should we have a policy about this?

Reply:
(oh, and what does nb mean in this context? I only know it as "nonbinary"^^)

Reply:
"nota bene", https://en.wikipedia.org/wiki/Nota_bene

Reply:
> Should we have a policy about this?

hm, we could add a line to the README, a "policy" seems a bit too much :)